client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,24 @@
 import React from 'react';
 
 const MovieCard = props => {
-  return;
+  return(
```

Comment:
Can put a statement right before this and have `const { title, director, metascore, stars } = props.movie;`. Then you can get rid of all the props and this will work. But need to change one more thing.